data/ab21.json

```
@@ -0,0 +1,102 @@
+{
+  "id": 244,
+  "name": "AB:21",
+  "tagline": "Liquorice Blackcurrant Stout.",
+  "first_brewed": "2016",
+  "description": "AB:21 is a liquorice and blackcurrant infused imperial stout. Rolling in at an imposing 12% abv, this huge and decadent dark beer dials down the charred roastiness and focuses instead on the rich, full-bodied chocolate and dark fruit character of the style.",
+  "image_url": "null",
```

This actually needs to be null rather than a string containing the characters "null"

---

Apologies. All set now. Found a few others from older recipes and fixed them up too,

---

HI there, this is still incorrect it needs to be like the suggestion below
```suggestion
  "image_url": null,
```

---

@samjbmason that causes the tests to fail. It needs to be a string.

`[ { keyword: 'type',
    dataPath: '.image_url',
    schemaPath: '#/properties/image_url/type',
    params: { type: 'string' },
    message: 'should be string' } ]
`

---

@samjbmason All fixed in https://github.com/samjbmason/punkapi-db/pull/41/commits/c7bd8e9a7223d73f1c5f7c12166d6a1d40017567, https://github.com/samjbmason/punkapi-db/pull/41/commits/fb8f0d7da520b6b5ca2bca07ed0a2f4577e91a64 and https://github.com/samjbmason/punkapi-db/pull/41/commits/5e3a7c3e4721aabb69338e84f88e0888193229bc